pom.xml

```diff
@@ -71,7 +71,29 @@
         </extensions>
 
         <plugins>
-                 
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <id>clone-repository</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>git</executable>
+                            <arguments>
+                                <argument>clone</argument>
+                                <argument>https://github.com/eclipse-uprotocol/uprotocol-core-api.git</argument>
+                                <argument>${project.build.directory}/uprotocol-core-api</argument>
```

Comment:
You need to pull a specific tag and not ToT

Reply:
Done